AEPTestUtils.podspec

```diff
@@ -11,11 +11,15 @@ Pod::Spec.new do |s|
   s.license          = { :type => "Apache License, Version 2.0", :file => "LICENSE" }
   s.author           = "Adobe Experience Platform SDK Team"
   s.source           = { :git => "https://github.com/adobe/aepsdk-testutils-ios.git", :tag => s.version.to_s }
-  s.ios.deployment_target = '10.0'
+  
+  s.ios.deployment_target = '11.0'
+  s.tvos.deployment_target = '11.0'
+
   s.swift_version = '5.1'
 
   s.pod_target_xcconfig = { 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
   s.dependency 'AEPCore', '>= 4.0.0'
```

Comment:
Should we add AEPServices as a dependency as well? I think Core depends on services so it should be downloaded nonetheless.